hlx_statics/blocks/embed/embed.js

```diff
@@ -57,6 +58,9 @@ const loadEmbed = (block, a) => {
     block.removeEventListener('mouseover', videoListener);
   };
   block.addEventListener('mouseover', videoListener);
+  decorateBackgroundColor(block);
+  const wid = block.parentElement.parentElement.getAttribute('data-width');
+  if (wid) block.firstChild.firstChild.style.width = wid;
```

Comment:
It looks like you are hard coding the behavior instead of using `decorateWidth` here. Can you check that this is the intended solution.

Reply:
yes, because the wid setting for this component is inside the child instead.